src/coreclr/src/jit/flowgraph.cpp

```diff
@@ -299,7 +299,7 @@ void Compiler::fgComputeProfileScale()
     //
     if (calleeWeight < callSiteWeight)
     {
-        JITDUMP("   ... callee entry count %d is less than call site count %d\n", calleeWeight, callSiteWeight);
+        JITDUMP("   ... callee entry count %f is less than call site count %f\n", calleeWeight, callSiteWeight);
```

Comment:
I wonder if it's worthwhile to introduce some `FMT_WEIGHT` define to ensure uniformity in dumping of weight data.

Reply:
Not sure. A lot of the dumping is already stylized (via `refCntWtd2str`).